packages/components/src/internal/components/assay/RunDataPanel.tsx

```diff
@@ -202,6 +205,17 @@ export class RunDataPanel extends PureComponent<Props, State> {
         this.props.onTextChange('');
     };
 
+    getEditableGridColumnMetadata = (): Map<string, EditableColumnMetadata> => {
+        const { wizardModel, plateSupportEnabled } = this.props;
+        if (!plateSupportEnabled) return undefined;
+
+        return Map({
+            Plate: {
+                lookupValueFilters: [Filter.create('PlateSet', wizardModel.runProperties?.get('PlateSet'))],
```

Comment:
added check for the existence of a selected plateSet. as mentioned, this would go away if we decide to remove the "Enter Data in Grid" tab for the plate metadata assay case.